setup.py

```diff
@@ -105,6 +105,9 @@ def get_git_sha():
         'sqlalchemy-utils',
         'sqlparse',
         'unicodecsv',
+        # Pinning werkzeug due to problems encountered with
+        # new config options added in 0.15.x
+        'werkzeug<0.15.0',
```

Comment:
@craig-rueda maybe it's worthwhile raising an issue in Flask. In seems in their (CHANGES](https://github.com/pallets/flask/blob/master/CHANGES.rst) in `mater` they've bumped the minimum Werkzeug version to >= 0.15 but maybe need to restrict existing Flask versions to `<0.15.0`.